packages/amplify-graphql-transformer-core/src/transformer-context/resolver.ts

```diff
@@ -272,8 +277,11 @@ export class TransformerResolver implements TransformerResolverProvider {
       this.responseMappingTemplate,
       this.datasource?.name || NONE_DATA_SOURCE_NAME,
       scope,
+      this.runtime,
     );
 
+    const { stashString, stashExpression } = this.generateStashStatementGenerator(this.runtime);
```

Comment:
nit: having two `generate`s in same name is confusing. Would rename this to `createStashStatementGenerator`.

Reply:
Good call. Addressed in [35481e8](https://github.com/aws-amplify/amplify-category-api/pull/2812/commits/35481e8689c9ba316c20adf2b32461d17486193f)